pyfcm/__meta__.py

```diff
@@ -1,10 +1,10 @@
-__title__ = 'pyfcm'
-__summary__ = 'Python client for FCM - Firebase Cloud Messaging (Android, iOS and Web)'
-__url__ = 'https://github.com/olucurious/pyfcm'
+__title__ = "pyfcm"
+__summary__ = "Python client for FCM - Firebase Cloud Messaging (Android, iOS and Web)"
+__url__ = "https://github.com/olucurious/pyfcm"
 
-__version__ = '1.5.2'
+__version__ = "1.5.2"
```

Comment:
```suggestion
__version__ = "1.5.5"
```
need to fix version string 1.5.5 or somthing